homeassistant/components/plex/__init__.py

```diff
@@ -246,6 +251,7 @@ def close_websocket_session(_):
     hass_data[DISPATCHERS][server_id].append(unsub)
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+    await hass.config_entries.async_forward_entry_setups(entry, [Platform.UPDATE])
```

Comment:
Why is this separate and not just added to the existing `PLATFORMS` constant?

Reply:
I was misremembering the intended startup flow. It definitely can be.